docs/contributor/weight-generation.md

```diff
@@ -19,51 +19,53 @@ In a PR run the actions through comment:
 
 To regenerate all weights (however it will take long,
 so don't do it unless you really need it), run the following command:
+
```

Comment:
This is a great doc, is it properly linked and discover-able from the main `CONTRIBUTING.md`?

Reply:
yep, there is a general link to [/cmd commands-readme](https://github.com/paritytech/polkadot-sdk/blob/master/docs/contributor/CONTRIBUTING.md#ui-tests) and from there to here